authz/middleware.go

```diff
@@ -54,7 +54,7 @@ func (mw *AuthzMiddleware) MiddlewareFunc(h rest.HandlerFunc) rest.HandlerFunc {
 		l := log.FromContext(r.Context())
 
 		//get token, no token header = http 401
-		tokstr, err := ExtractToken(r.Request)
+		tokstr, err := ExtractToken(r.Request) // here
```

Comment:
what's this comment about?

Reply:
fixed. sorry for that, this is how I debug :)